app/src/layout/Login.jsx

```diff
@@ -20,12 +25,10 @@ export default function Login() {
   }, []);
 
   return (
-    <Menu as='div' className='relative my-auto ml-5 z-50'>
-      <div>
-        <Menu.Button className='relative inline-flex items-center rounded-md border border-transparent bg-secondary px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-primary focus:outline-none focus:ring-2 focus:ring-accent3 focus:ring-offset-2 focus:ring-offset-gray-800'>
-          <span>{currentUserType || "Login"}</span>
-        </Menu.Button>
-      </div>
+    <Menu as='div' className='relative my-auto ml-5'>
```

Comment:
Can we not entirely remove the `z-50` here? Perhaps lowering the value such that it still outweighs the modal?

![image](https://user-images.githubusercontent.com/84013254/199202812-a7fe4aee-99da-414b-8e5e-badc53f1ff3a.png)
Create job ([link](https://github.com/is212g7t5/is212g5t7-project/blob/1951cda4255ec79ebf6fe0f70fd170408c92baab/app/src/components/job/hr/CreateJob.jsx#L43)) seems to be blocking the user selection

Reply:
Oh man... Ok let me change it.

Reply:
Changed the login z index to 10 and modal to 50.